### PR TITLE
Limit loc string to avoid header overflowing and other minor fixes

### LINKED ISSUE #1537: Location in Message Reader wraps around and overflows
If Location of system exceeds the screen width, the header wraps around and overflows. This problem appears in Traditional and Full Screen Readers.

Screenshot from 2023-01-22 03-23-18
Screenshot from 2023-01-22 03-22-51

### PR DESCRIPTION
**Pull request by ericpareja**
Fixes #1537

### CODE REVIEW

File: wwivutil/messages/messages.cpp [hunk 1]
```diff
@@ -241,7 +242,7 @@ class PostMessageCommand final : public BaseMessagesSubCommand {
 
     MessageAreaOptions area_options{};
     area_options.send_post_to_network = true;
-    area_options.add_re_and_by_line = true;
+    area_options.add_re_and_by_line = arg("use_re_and_by").as_bool();
```
Comment (ericpareja) [inline wwivutil/messages/messages.cpp:245]: added use_re_and_by switch as an option.

File: bbs/xferovl1.cpp [hunk 2]
```diff
@@ -480,7 +480,7 @@ void tag_files(bool& need_title) {
       bout.clear_lines_listed();
       a()->filelist.clear();
       need_title = true;
-      bout.cls();
+      //bout.cls();
```
Comment (wwiv) [inline bbs/xferovl1.cpp:483]: why?

File: wwivutil/messages/messages.cpp [hunk 3]
```diff
@@ -164,15 +164,16 @@ class PostMessageCommand final : public BaseMessagesSubCommand {
   PostMessageCommand() : BaseMessagesSubCommand("post", "Posts a new message.") {}
 
   bool AddSubCommands() override {
-    add_argument({"title", "message sub name to post on", ""});
-    add_argument({"from", "message sub name to post on", ""});
-    add_argument({"from_usernum", "message sub name to post on", ""});
-    add_argument({"to", "message sub name to post on", ""});
-    add_argument({"date", "message sub name to post on", ""});
-    add_argument({"in_reply_to", "message sub name to post on", ""});
+    add_argument({"title", "title of post", ""});
```
Comment (wwiv) [inline wwivutil/messages/messages.cpp:167]: oh thanks!

**Review by wwiv — APPROVED** [review]

File: bbs/xferovl1.cpp (hunk 2 above)
Comment (ericpareja) [reply bbs/xferovl1.cpp:483]: oh.. in transfer section, when in non-ANSI mode, when you L)ist files, you get the File Tagging prompt and Q)uitting didn't clear the screen, whereas C)ontinue, SPACE or RETURN would. It wasn't a consistent experience. I did tests with it and not clearing the screen made more sense. an option i was thinking of was to make  bout.cls() behavior in spots like these follow the D)efaults S) Clear screen between messages switch. Clear the screen if that toggle is on.

Comment (wwiv) [reply bbs/xferovl1.cpp:483]: that sounds great to have it follow an option like that.

File: wwivutil/messages/messages.cpp (hunk 3 above)
Comment (ericpareja) [reply wwivutil/messages/messages.cpp:167]: i need clarification on what in_reply_to refers to though. what are the expected possible values here?

Comment (wwiv) [reply wwivutil/messages/messages.cpp:167]: from message_area_wwiv.cpp

    } else if (starts_with(line, "RE:")) {
      r.in_reply_to = StringTrim(line.substr(3));
   ....

It's just the "RE: XXXXX" line text without the "RE: "

**Pull request merged**